### ISSUE TITLE
Hydra build may fail because of the state of the current hydra master

### ISSUE THREAD

**Issue by datakurre** [peti/hydra-tutorial#1]
For those trying this out and getting a build failure, this was the latest revision working for me today (2016-03-28):

```
let

  hydraSrc = builtins.fetchTarball https://github.com/nixos/hydra/archive/993647d1e3b43f1f9b7dc2ebce889b475d156bb9.tar.gz;
```

**Comment by peti** [comment]
Yeah, according to http://hydra.nixos.org/jobset/hydra/master, 9a7b24a is the last revision that built successfully for the time being. Hopefully this gets fixed soon.

**Comment by domenkozar** [comment]
Hopefully it's fixed with https://github.com/NixOS/nixpkgs/commit/8e398a88a1fb3f477667d8bf42e0d8f1d8d5ddec

**Comment by domenkozar** [comment]
I'm now running Hydra from master branch using the above commit. This can be closed :)

PS: maybe we should warn users to try older commits if master is somehow broken?

**Comment by peti** [comment]
Seems like hydra is broken again: http://hydra.nixos.org/jobset/hydra/master. :disappointed:

**Comment by domenkozar** [comment]
I hope this will be soon resolved now that Hydra is part of nixpkgs again: https://github.com/NixOS/nixpkgs/commit/3e631800d1ddc93523be4a3a6880a33dc80efb2e

See https://github.com/peti/hydra-tutorial/pull/7
